tests/scripts/all.sh

```diff
@@ -1479,6 +1480,7 @@ component_test_full_no_cipher () {
     scripts/config.py unset MBEDTLS_SSL_TICKET_C
     # Indirect dependencies
     scripts/config.py unset MBEDTLS_SSL_CLI_C
+    scripts/config.py unset MBEDTLS_PSA_CRYPTO_CONFIG
```

Comment:
This component disables PSA, so that line has no effect anyway. But I'll keep it, since [we're close to being able to enable PSA here](https://github.com/Mbed-TLS/mbedtls/issues/8048), and then the line will be needed.